gnuradio-runtime/include/gnuradio/rpcregisterhelpers.h

```diff
@@ -802,8 +802,8 @@ struct rpcbasic_register_set : public rpcbasic_base {
      * \param display_     The display mask
      */
     rpcbasic_register_set(const std::string& block_alias,
-                          const char* functionbase,
-                          void (T::*function)(Tto),
+                          __GR_ATTR_UNUSED const char* functionbase,
```

Comment:
C++17 introduced [[[maybe_unused](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)]] that we use elsewhere for variables

I see __GR_ATTR_UNUSED in our codebase for functions that are not used, maybe this should become `[[maybe_unused]]`here for consistency

Reply:
Oh, nice! I guess I can just convert everything over to `[[maybe_unused]]` in this PR then.